config/provisioners/in-memory-channel/in-memory-channel.yaml

```diff
@@ -97,12 +97,7 @@ rules:
     resources:
       - events
     verbs:
```

Comment:
Does this work? I thought you tried with only the create permission and it failed.

Reply:
I tried it yesterday and it did. I just tried again and I cannot see events. Maybe I did something wrong yesterday. Let me add the patch verb and try again.

Reply:
I think the logs will say what permission is missing.

Reply:
I just applied the in-memory-channel with just create permission, deleted the in-memory-channel-controller pod (it got re-created).. If I ask to describe the events, I can see them. I'm not sure if I was applying the patch properly before. 
It seems to be working

<img width="787" alt="screen shot 2019-01-24 at 12 38 57 pm" src="https://user-images.githubusercontent.com/4714124/51707309-b5cb3780-1fd5-11e9-9b0c-5e4ed3403e38.png">
<img width="1059" alt="screen shot 2019-01-24 at 12 38 42 pm" src="https://user-images.githubusercontent.com/4714124/51707314-b82d9180-1fd5-11e9-8cbc-2b10fa169614.png">
<img width="953" alt="screen shot 2019-01-24 at 12 38 30 pm" src="https://user-images.githubusercontent.com/4714124/51707317-ba8feb80-1fd5-11e9-8eb4-74aa9ef33383.png">